tests/DependencyInjection/ClassNotFoundException.php

```diff
@@ -0,0 +1,22 @@
+<?php
+declare(strict_types=1);
+
+namespace Lcobucci\Chimera\Tests\DependencyInjection;
+
+final class UnableToLoadPackage extends \RuntimeException
+{
+    const ERROR_MESSAGE = 'Application cannot be loaded because: %s, is not defined in you application.';
+
+    const COMMAND_BUS_SERVICE_CLASS = 'Lcobucci\Chimera\Bus\Tactician\DependencyInjection';
+    const ROUTING_SERVICE_CLASS = 'Lcobucci\Chimera\Routing\Expressive\DependencyInjection';
+
+    public static function forMissingBusService(): self
```

Comment:
Since we might have different packages which plugs other service buses or routing libraries, this seems better to me:

```php
public static function serviceBusMissing(string $class): self
{
    return new self(sprintf(self::ERROR_MESSAGE, $class));
}

public static function routingMissing(string $class): self
{
    return new self(sprintf(self::ERROR_MESSAGE, $class));
}
```

Then we also drop the constants...